tests/waterservices_test.py

```diff
@@ -321,10 +323,10 @@ def test_get_discharge_measurements(requests_mock):
 @pytest.mark.parametrize("site_input_type_list", [True, False])
 def test_get_discharge_measurements_sites_value_types(requests_mock, site_input_type_list):
     """Tests get_discharge_measurements method for valid input types for 'sites' parameter"""
-    _format = "rdb"
```

Comment:
Typically, it implies that a variable is intended for internal use; i.e. it's used by functions but not directly by the user.
However, in this case, I don't understand why this was done, unless to avoid some naming conflict, but I don't see it.
I'd leave it or now, but sometime it would be good to go back and clean and lint all the tests.